sourcecred/template-instance
# Improve user experience for setting up environment variables

Setup dotenv to load environment variables from a `.env` file instead of assuming the user
knows how to set environment variables in their shell on their own. Updated the README instructions
to make it much more clear and explicit for how users should set their environment variables.

Test Plan: Without any environment variables set:
1. Enable the "sourcecred/github" plugin
2. Add your github token to the `.env` file
3. Run `yarn load` and ensure that it successfully reads the token and loads github data into the cache.
